projects/storefrontlib/src/cms-components/configurator/textfield/textfield-configurator.module.ts

```diff
@@ -50,12 +51,14 @@ import { ConfigTextfieldInputFieldComponent } from './config-textfield-input-fie
   declarations: [
     ConfigTextfieldFormComponent,
     ConfigTextfieldInputFieldComponent,
+    ConfigTextfieldAddToCartButtonComponent,
   ],
   exports: [ConfigTextfieldFormComponent, ConfigTextfieldInputFieldComponent],
```

Comment:
Missing component. ConfigTextfieldAddToCartButtonComponent should be exported as well

Reply:
done